polygeist/include/mlir/Dialect/Polygeist/IR/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
 add_mlir_dialect(PolygeistOps polygeist)
-add_mlir_doc(PolygeistOps PolygeistOps Dialects/ -gen-dialect-doc -dialect=polygeist)
+add_mlir_doc(PolygeistBase PolygeistBase Dialects/ -gen-dialect-doc)
+add_mlir_doc(PolygeistOps PolygeistOps Dialects/ -gen-op-doc)
+add_mlir_doc(PolygeistTypes PolygeistTypes Dialects/ -gen-typedef-doc)
```

Comment:
Is the documentation being generated correctly? (should be target `polygeist-doc`)

Reply:
The documentation is now being generated correctly.